packages/manager/apps/telecom/src/app/telecom/telephony/carrierSip/dashboard/dashboard.routing.js

```diff
@@ -21,6 +21,12 @@ export default /* @ngInject */ ($stateProvider) => {
           billingAccount,
           serviceName,
         }),
+      clusterDetails: /* @ngInject */ ($http, billingAccount, serviceName) =>
+        $http
+          .get(
+            `/telephony/${billingAccount}/carrierSip/${serviceName}/clusterDetails`,
+          )
+          .then(({ data }) => data),
```

Comment:
Is that make sense for you to use a `catch` block?
To prevent blocking access to the dashboard I mean.